### PR TITLE
new about page layout

### PR DESCRIPTION
**Pull request by Viktorika-p**

### CODE REVIEW

**Review by MetalMari — APPROVED** [review]

File: front-end/src/App.js [hunk 1]
```diff
@@ -2,12 +2,15 @@ import './App.css';
...
 import Auth_Header from './Components/Header/Auth_Header';
 import Header from '../src/Components/Header/Header';
+import { Route } from 'react-router-dom';
+import Footer from './Components/Footer/Footer';
 
 function App() {
   return (
     <>
       <Auth_Header />
       {/* <Header /> */}
```
Comment (vvelizariy) [inline front-end/src/App.js:16]: Do you need this commented line?

File: front-end/src/Components/Header/Navigation.js [hunk 2]
```diff
@@ -8,7 +9,7 @@ function Navigation () {
                 <li><a href="/about">About</a></li>
                 <li><a href="/news">News</a></li>
                 <li><a href="/contacts">Contacts</a></li>
-                <li><a href="/jobs">Jobs</a></li>
+                <li><a href="/jobs">Jobs</a></li>
```
Comment (vvelizariy) [inline front-end/src/Components/Header/Navigation.js:12]: remove whitespaces after tag

File: front-end/src/Components/pages/aboutPage/AboutPage.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { Carousel } from 'react-bootstrap';
+import './aboutpage.css';
```
Comment (vvelizariy) [inline front-end/src/Components/pages/aboutPage/AboutPage.js:3]: should be used camelCase naming

File: front-end/src/Components/pages/aboutPage/AboutPage.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            <Carousel.Item>
+                <section className="teamtest">
+                    <h2>Our team</h2>
+                    <div>
+                        <ol>
+                            <li><a href="/">Intern's Softserve - authors</a></li>
+                            <li><a href="/">Viktoriia Zakharko - moderathor</a></li>
+                            <li><a href="/">Sofiia Tkachuck - moderathor</a></li>
+                            <li><a href="/">Oksana Lisovska - moderathor</a></li>
+                            <li><a href="/">Dmytro Baryshnuik - moderathor</a></li>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/aboutPage/AboutPage.js:34]: typo in `moderathor word`

File: front-end/src/Components/pages/aboutPage/AboutPage.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                </section>
+                <Carousel.Caption>
+                </Carousel.Caption>
+            </Carousel.Item>
+            <Carousel.Item>
+                <section className="teamtest">
+                    <h2>Our team</h2>
+                    <div>
+                        <ol>
+                            <li><a href="/">Intern's Softserve - authors</a></li>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/aboutPage/AboutPage.js:30]: better to move users to list of objects, then use map to iterate and create li elements user - position.
But for initial page is ok.

**Review by okslisovska — APPROVED** [review]

File: front-end/src/App.js (hunk 1 above)
Comment (Viktorika-p) [reply front-end/src/App.js:16]: Yes, because we have two headers, but functionality for their changes don't added yet.

**Review by s0fko — APPROVED** [review]

**Pull request merged**